INSTALL.md

```diff
@@ -75,7 +75,15 @@ working for other remotes or architectures. For this example, we'll refer to the
 development and have your kernel sources available as the `development machine` and the machine you're tracing as the
 `target`.
 
+#### Install build dependencies
+```
+sudo apt-get install cmake libelf-dev bison flex iperf netperf
+```
+
 #### Build LLVM on your development host
+First, remove any existing installation of LLVM and clang from your system.
```

Comment:
removal of existing LLVM is not really needed. All one has to do is, before they build BCC, just point `PATH` to the custom built LLVM in its build bin directory.
I just do something like this before building BCC:
`export PATH=/path/to/llvm/build/bin:$PATH`

As long as it find llvm-config from there (do a `which llvm-config` to confirm), it will pick up the correct clang/llvm.

Reply:
Okay, removed. Done.